charts/asset-manager/templates/clamav-configmap.yaml

```diff
@@ -9,29 +9,27 @@ metadata:
     argocd.argoproj.io/hook: PreSync
 data:
   clamd.conf: |
+    # If it's too big to scan, fail it.
+    AlertExceedsMax yes
+    # Avoid RAM spikes on reload. Scanning tasks are async anyway.
+    ConcurrentDatabaseReload no
+    ExtendedDetectionInfo yes
+    Foreground yes
     LogTime yes
-    LogSyslog yes
```

Comment:
There are quite a few changes happening in the same commit, which would make it trickier to revert if any one of them is an issue. Could you split out each bulletpoint into its own commit?